types/gulp-rename/index.d.ts

```diff
@@ -4,6 +4,7 @@
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
 
 /// <reference types="node"/>
+import { VinylFile } from "gulp-typescript/release/types";
```

Comment:
I think this would be better as `import { File } from 'vinyl';` instead, which will reference the type definition in `@types/vinyl`

Reply:
Thanks. Done.